src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,273 @@
+
+<DocumentList>
+<DocumentListItem
+    title="Select a Builder"
+    subtitle="Learn how to select a Builder for your Workspace and customize your development environment."
+    href="/usage/builders"
+    expanded
+  />
+</DocumentList>
+
+## Configuring Daytona Server
+
+The [Daytona Server](/usage/server) is a crucial component that runs as a daemon on your machine, responsible for managing [Workspaces](/usage/workspaces), [Providers](/configuration/providers), and other system configurations. The Daytona Server provides a centralized interface for interacting with the Daytona platform, allowing you to manage your development environments. The Daytona Server can be easily managed by using the [Daytona CLI](/reference/cli).
+
+Daytona offers flexible configuration options for the server. The configuration file, which contains settings such as the server ID, API port, and default project image, is located in different directories depending on your operating system:
+
+- On macOS, the configuration file is located at:
+
+    `~/Library/Application\ Support/daytona/server/config.json`.
+
+- On Linux, the configuration file is located at:
+
+    `~/.config/daytona/server/config.json`.
+
+- On Windows, the configuration file is located at:
+
+    `C:\Users\YOUR_USERNAME\AppData\Roaming\daytona\server\config.json`.
+
+To view the Daytona Server configuration, use the `daytona server config` command. This command displays the server configuration properties, allowing you to manage and customize the server settings as needed.
+
+Daytona allows you to edit the server configuration directly by using the `daytona server configure` command. This command opens the configuration file, enabling you to modify the server settings based on your requirements.
+
+<DocumentList>
+<DocumentListItem
+    title="Configure the Daytona Server"
+    subtitle="Learn how to configure the Daytona Server and manage server settings."
+    href="/usage/server"
+    expanded
+  />
+</DocumentList>
+
+## Using the CLI
+
+The Daytona Command-Line Interface (CLI) is the primary method of interacting with the Daytona platform. The CLI provides a comprehensive set of commands that allow you to manage Workspaces, Providers, and other system configurations.
+
+To access the Daytona CLI, open your terminal and use the `daytona` command followed by the desired subcommand. The CLI offers a wide range of commands, each serving a specific purpose, such as creating Workspaces, managing Providers, setting Targets, and configuring system settings.
+
+For a complete list of available commands and their descriptions, refer to the [CLI Reference](/reference/cli). This reference guide provides detailed information on each command, including usage examples and command options, helping you leverage the full potential of the Daytona CLI.
```

Comment:
I would mention that users can run `daytona --help` to list all commands as well